src/components/tiles/table/cell-text-editor.tsx

```diff
@@ -51,37 +48,61 @@ export default function CellTextEditor<TRow, TSummaryRow = unknown>({
     _column.appData?.onEndBodyCellEdit?.(newValue);
   };
 
-  const finishAndSave = () => {
-    const endValue = valueRef.current;
-    if (endValue !== origValueRef.current) {
-      onRowChange({ ...row, [column.key]: endValue }, true);
+  const finishAndSave = (commitChanges: boolean) => {
+    if (commitChanges) {
+      const endValue = valueRef.current;
+      if (endValue !== origValueRef.current) {
+        onRowChange({ ...row, [column.key]: endValue }, true);
+      }
```

Comment:
FYI, I learned recently while working in this area that `onRowChange(..., false)` while editing maintains a local copy of the edit state which is then flushed by `onClose(true)`. In other words, I think you could also call `onRowChange(...)` in `updateValue()` and then you wouldn't need to call it here because `onClose(true)` will commit the value. I believe this would be a more in line with the built-in implementation and would potentially prevent a double-commit under the hood, but it may not make any difference in the end.

Reply:
Updated as suggested.